### ISSUE TITLE
Train on custom 2D pose dataset

### ISSUE THREAD

**Issue by alexanderfrey** [Arthur151/ROMP#92]
Hi and thanks for great work !

I would like to finetune the model on some custom 2D pose dataset. From what I read you trained on 3D and 2D pose datasets so that should be possible, right ?

Best
Alexander

**Comment by Arthur151** [comment]
Yes, of course. You can refer to [this issue](https://github.com/Arthur151/ROMP/issues/88) to create your own dataset.

**Comment by alexanderfrey** [comment]
Thanks for your answer. That particular issue is about retraining on the model output. My question is more related to how the data in the img_info dictionary is generated and what is the format ? 

I have right now 2D pose data in a csv file with the format `nose,182,416,img001.jpg,768,800` etc.

How do I go from that data to the img_info dict ?

Any help is very much appreciated.

Best,
Alexander

**Comment by Arthur151** [comment]
[Crowdpose](https://github.com/Arthur151/ROMP/blob/master/romp/lib/dataset/crowdpose.py) is a perfect example for your need. It just loads the 2D pose data. You need to replace [this line](https://github.com/Arthur151/ROMP/blob/992544341d9469c30de884403a3fc7b977974aa8/romp/lib/dataset/crowdpose.py#L31) to the 2D pose of your data. Please be sure to set the correct keypoint ordering at [this line](https://github.com/Arthur151/ROMP/blob/992544341d9469c30de884403a3fc7b977974aa8/romp/lib/dataset/crowdpose.py#L11). To achieve this, you can define a dict in constants.py like [this](https://github.com/Arthur151/ROMP/blob/992544341d9469c30de884403a3fc7b977974aa8/romp/lib/constants.py#L139). Then you can replace constants.Crowdpose_14 to the keypoint order dict you defined.
self.kp2d_mapper = constants.joint_mapping(constants.KEYPOINT_ORDER,constants.SMPL_ALL_54)
